apps/core/factories.py

```diff
@@ -53,13 +53,13 @@ class Meta:
 
 class PixelerFactory(DjangoModelFactory):
 
-    date_joined = Faker('date_object')
+    date_joined = Faker('date')
```

Comment:
Like mentioned in my previous review: this is not a `date` but a `datetime`. See Faker documentation to choose the appropriate provider: https://faker.readthedocs.io/en/master/providers/faker.providers.date_time.html

Reply:
Take time to check model fields types so that you choose relevant Faker providers.